skaha/src/main/java/org/opencadc/skaha/session/Session.java

```diff
@@ -83,6 +83,7 @@ public class Session {
     
     private String id;
     private String userid;
+    private String appid;
```

Comment:
I guess appId is null for regular sessions.  Do you think it would be worth it to have a hierarchy of types?  Abstract superclass 'job' (or something) with 'session' and 'desktop-app' as subclasses?  Not sure...

Reply:
Will postpone this work to Session modelling in another story.